doc/api/os.md

```diff
@@ -378,9 +378,8 @@ added: v0.3.3
 
 The `os.uptime()` method returns the system uptime in number of seconds.
 
-*Note*: Within Node.js' internals, this number is represented as a `double`.
-However, fractional seconds are not returned and the value can typically be
-treated as an integer.
+*Note*: the return value includes fractions of a second on Windows.
```

Comment:
> we should be exact.

@gibfahn I agree in principle, can we figure this out exactly? Until recently we had a hard time list all supported platform (#11872). Mentioning Windows has an implicit meaning that it's only on Windows. And as @bnoordhuis said

> UNIX implementations of that function don't currently bother to mix in the sub-second fraction. That could change in the future though, if someone requests it.